### PR TITLE
PD-2253: Removed m3c Dockerfiles from warp

### PR DESCRIPTION
**Pull request by aawdeh**
Moved Dockerfiles from warp to warp-tools. Need to remove Dockerfiles from warp.

Link to Dockerfiles in warp: https://github.com/broadinstitute/warp-tools/tree/develop/3rd-party-tools/m3c-yap-hisat

PR related to ticket PD-2253. 

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [x] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Review by jessicaway — APPROVED** [review]

**Review by ekiernan — CHANGES_REQUESTED** [review]
@aawdeh Did you move the hisat3-build-indices Docker to warp-tools? I didn't see it, and we had discussed leaving it in warp until we confirm we have the indices we need for BICAN analysis. Just wanted to check before merging.

File: beta-pipelines/skylab/m3c/Dockers/hisat-3n-build-index-docker/Dockerfile
Comment (ekiernan) [inline beta-pipelines/skylab/m3c/Dockers/hisat-3n-build-index-docker/Dockerfile:1]: I think we want to keep this here?

Comment (aawdeh) [reply beta-pipelines/skylab/m3c/Dockers/hisat-3n-build-index-docker/Dockerfile:1]: No I didn't move it. I thought we were going to remove it -- I asked @nikellepetrillo about this file and she said that we don't need it. The contents of the hisat3-build-indices Dockerfile are found in the m3c-yap-hisat Dockerfile.

Comment (aawdeh) [reply beta-pipelines/skylab/m3c/Dockers/hisat-3n-build-index-docker/Dockerfile:1]: I can move it as is to warp-tools if its needed tho

Comment (aawdeh) [reply beta-pipelines/skylab/m3c/Dockers/hisat-3n-build-index-docker/Dockerfile:1]: Or keep it here

Comment (ekiernan) [reply beta-pipelines/skylab/m3c/Dockers/hisat-3n-build-index-docker/Dockerfile:1]: Ok, thanks for clarifying! You're right, if we need we'll be able to build again based on the other Docker.

**Review by ekiernan — APPROVED** [review]

File: beta-pipelines/skylab/m3c/Dockers/hisat-3n-build-index-docker/Dockerfile
Comment (aawdeh) [reply beta-pipelines/skylab/m3c/Dockers/hisat-3n-build-index-docker/Dockerfile:1]: Ok awesome, thanks for the thumbs up!

**Pull request merged**